ts/Series/Column/ColumnSeries.ts

```diff
@@ -494,6 +494,13 @@ class ColumnSeries extends Series {
         const series = this,
             chart = series.chart,
             options = series.options,
+            // For points whithout graphics (null points) this value is used
+            // to reserve space around the point such that:
+            //      - normal/null points are spaced similarily,
+            //      - focusborders of null points are like those of "0" points
+            // This ensures consistent dimensions between null/normal points.
+            minimumSpace = 8,
+            nullMinSpace = options.nullInteraction && minimumSpace || 0,
```

Comment:
No need for a stepping stone?
```suggestion
            nullMinSpace = options.nullInteraction ? 8 : 0,
```